### PR TITLE
Added python-dotenv to make running locally easier

### PR DESCRIPTION
**Pull request by cronym**
Added support for python-dotenv to keep creds in a local .env file because I always forget to export.

### CODE REVIEW

**Comment by jcsalterego** [comment]
Thanks for the contribution! Have you tested it in GitHub actions by any chance?

**Comment by cronym** [comment]
I haven't, but it shouldn't interfere with the previous workflow in any way. If there's not an .env file, it does nothing.

I can set up a test run in a bit to verify, though!

**Comment by jcsalterego** [comment]
No worries @cronym - I think I'm missing a `pip install` which I will get in there before merging this.

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 requests==2.28.2
 black==23.3.0
 ruff==0.0.262
+python-dotenv=1.0.0
```
Comment (jcsalterego) [inline requirements.txt]: Looks like this might need to be `==`

**Review by jcsalterego — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (cronym) [reply requirements.txt]: whoops, that's my bad. I haven't used `requirements.txt` in years. I'll fix that.

**Comment by jcsalterego** [comment]
Perfect, thanks! Be sure to grab the latest `main` if you plan to run it in GitHub actions.

**Pull request merged**